src/Win32/Win32Enums.cs

```diff
@@ -6,7 +6,7 @@ namespace Axe.Windows.Win32
     /// <summary>
     /// https://msdn.microsoft.com/en-us/library/windows/desktop/ms646309(v=vs.85).aspx
     /// </summary>
-    public enum HotkeyModifier : int
+    internal enum HotkeyModifier : int
```

Comment:
Not necessary for this PR, but HotkeyModifier can safely be removed as it's only used in AIWin (which has its own version)